### ISSUE TITLE
Unreal crash when drawing 3D Tiles

### ISSUE THREAD

**Issue by bertt** [CesiumGS/cesium-unreal#886]
Hi,

We are trying to visualize 3D Tileset https://fileserv.beta.geodan.nl/test/unreal_test/tileset.json in Unreal 5.0.2 with Cesium plugin 1.14.0
but there is a crash when Unreal tries to draw.

Stacktrace:

```
Unhandled Exception: 0xe06d7363

KERNELBASE
VCRUNTIME140
msvcp140
UnrealEditor_CesiumRuntime!std::_String_val<std::_Simple_types<char> >::_Xran() [C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.29.30133\INCLUDE\xstring:2337]
UnrealEditor_CesiumRuntime!Cesium3DTilesSelection::Tile::processLoadedContent()
UnrealEditor_CesiumRuntime!Cesium3DTilesSelection::Tileset::_visitTileIfNeeded()
UnrealEditor_CesiumRuntime!Cesium3DTilesSelection::Tileset::_visitTile()
UnrealEditor_CesiumRuntime!Cesium3DTilesSelection::Tileset::_visitTileIfNeeded()
UnrealEditor_CesiumRuntime!Cesium3DTilesSelection::Tileset::updateView()
UnrealEditor_CesiumRuntime!ACesium3DTileset::Tick() [D:\build\U5M-Marketplace\Sync\LocalBuilds\PluginTemp\HostProject\Plugins\CesiumForUnreal\Source\CesiumRuntime\Private\Cesium3DTileset.cpp:1631]
UnrealEditor_Engine
UnrealEditor_Engine
UnrealEditor_Engine
UnrealEditor_Engine
UnrealEditor_Core
UnrealEditor_Core
UnrealEditor_Core
UnrealEditor_Engine
UnrealEditor_Engine
UnrealEditor_Engine
UnrealEditor_Engine
UnrealEditor_UnrealEd
UnrealEditor_UnrealEd
UnrealEditor
UnrealEditor
UnrealEditor
UnrealEditor
UnrealEditor
UnrealEditor
kernel32
ntdll
```

So it looks like there is something going wrong in D:\build\U5M-Marketplace\Sync\LocalBuilds\PluginTemp\HostProject\Plugins\CesiumForUnreal\Source\CesiumRuntime\Private\Cesium3DTileset.cpp:1631

In Cesium the tileset is visualized correctly: https://fileserv.beta.geodan.nl/test/unreal_test/index.html

![image](https://user-images.githubusercontent.com/538812/176135553-62a4ac52-8b14-4c6e-b891-d445acc65c66.png)

**Comment by bertt** [comment]
Update: I've tried to simplify the issue by removing from the glTF custom vertex attributes, Draco compression and trying other renderers but in all cases Unreal keeps crashing. 

I've got the impression there is something in the glTF what triggers the crash, although the glTF's are valid without warnings (https://github.khronos.org/glTF-Validator/)

**Comment by bertt** [comment]
A minimal reproducible case:

- Cesium viewer: https://bertt.github.io/cesium_unreal_samples/crash

![image](https://user-images.githubusercontent.com/538812/176212679-24714132-5720-46d8-ad4c-3f5055b27a9f.png)

- Tileset.json: https://bertt.github.io/cesium_unreal_samples/crash/tileset.json

- Glb: https://bertt.github.io/cesium_unreal_samples/crash/crash.glb

When loading tileset.json in Unreal there is a crash.

Sample wich does not crash in Unreal: https://bertt.github.io/cesium_unreal_samples/nocrash/

**Comment by kring** [comment]
@bertt thanks for reporting this issue. We've included a fix for it in v1.15.0, which was released at the end of last week.
